### PR TITLE
Add css class to make stat var hierarchy screenshot stable

### PR DESCRIPTION
**Pull request by shifucun**
Also make place page sources stable.

### CODE REVIEW

File: server/webdriver/tests/timeline_test.py [hunk 1]
```diff
@@ -150,9 +150,9 @@ def test_check_statvar_and_uncheck(self):
         (By.CLASS_NAME, 'dc-chart-exist'))
     WebDriverWait(self.driver, self.TIMEOUT_SEC).until(element_present)
 
-    # Assert there is one chart.
+    # Assert there is one chart and on opened hierarchy.
     charts = self.driver.find_elements(By.CLASS_NAME, 'dc-chart-exist')
-    self.assertEqual(len(charts), 1)
+    self.assertEqual(len(charts), 2)
```
Comment (chejennifer) [inline server/webdriver/tests/timeline_test.py:155]: nit: I think it's a little confusing that we are waiting on "dc-chart-exist" when waiting on stat var hierarchy to open .. Maybe we can either:

a. change the classnames to something more generic like dc-element-loaded & dc-container
b. have a separate classname for sv hierarchy exists

**Review by chejennifer — APPROVED** [review]
nice! Just one suggestion

File: server/webdriver/tests/timeline_test.py (hunk 1 above)
Comment (shifucun) [reply server/webdriver/tests/timeline_test.py:155]: Good idea. Renamed with generic names.

**Review by chejennifer — APPROVED** [review]
thanks for updating!

**Pull request merged**